tensorflow/python/kernel_tests/array_ops/array_ops_test.py

```diff
@@ -1923,6 +1923,19 @@ def test_quantize_and_dequantize_v4_grad():
       test_quantize_and_dequantize_v4_grad()
 
 
+class QuantizeAndDequantizeV2Test(test_util.TensorFlowTestCase):
+
+  def testInvalidArgs(self):
+    with self.assertRaisesRegex((errors.InvalidArgumentError, ValueError),
+                                r"(.*should be scalar.*|.*Broadcast dimension -1 is out of bound.*)"):
```

Comment:
This triggers completely different errors in TF vs XLA.  I think it would be better to remove the -1 from the test so that both error out for the same reason.

Reply:
@cantonios  Sorry for the late reply. I have update the test to remove the `-1`.